lib/jekyll/frontmatter_defaults.rb

```diff
@@ -130,7 +130,7 @@ def valid?(set)
     # new_scope - the new scope hash
     #
     # Returns true if the new scope has precedence over the older
-    def has_precedence?(old_scope, new_scope)
+    def precedence?(old_scope, new_scope)
```

Comment:
For all my source, I disable Rubocops "worthless private" warning and add private above every private def no matter how many there are, so it's clear that method is private, the same way other languages enforce that kind of explicitness.

```ruby
private
def method1
  #
end

private
def method2
  #
end
```